File: digit_ops/digit_ops.go
package main

import (
	"fmt"
)

/**
 * Вычисляет сумму и произведение цифр числа, а также модуль их разности.
 * 
 * @param num Натуральное число для обработки
 */

func calculateDigitOps(num int) {
	sum, product := 0, 1
	original := num
	
	for num > 0 {
		digit := num % 10
		sum += digit
		product *= digit
		num /= 10
	}
	
	diff := sum - product
	if diff < 0 {
		diff = -diff
	}
	fmt.Printf("%d %d %d\n", sum, product, diff)
}

func main() {
	var N int
	fmt.Scan(&N)
	calculateDigitOps(N)
}

File: digit_ops/DigitOps.java
import java.util.Scanner;

public class DigitOps {
    /**
     * Вычисляет сумму и произведение цифр числа, а также модуль их разности.
     * 
     * @param num Натуральное число для обработки
     */
    public static void calculateDigitOps(int num) {
        int sum = 0, product = 1;
        int original = num;
        
        while (num > 0) {
            int digit = num % 10;
            sum += digit;
            product *= digit;
            num /= 10;
        }
        
        System.out.println(sum + " " + product + " " + Math.abs(sum - product));
    }
    
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        calculateDigitOps(N);
        scanner.close();
    }
}